cvat/apps/annotation/cvat_ddln.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2020
+
+format_spec = {
+    "name": "DDLN_CSV",
+    "dumpers": [
+        {
+            "display_name": "{name} {format} {version} for images (BB)",
+            "format": "ZIP",
+            "version": "0.1",
+            "handler": "dump_ddln_csv"
+        }
+    ],
+    "loaders": [
+        {
+            "display_name": "{name} {format} {version}",
+            "format": "CSV",
+            "version": "0.1",
+            "handler": "load",
+        }
+    ],
+}
+
+def extractFileParams(filename):
+    directoryName = filename.split("/")[2]
+    csvFilename = filename.split("/")[-1]
+    csvFilename = csvFilename[:-4] + "_y_bb.csv"
+	
+    return directoryName,csvFilename
+
+def writeToCsv(dirname, filename, data):
+    try:
+        if not os.path.exists(dirname):
+            os.mkdir(dirname)
+        with open(dirname + filename, 'w', newline='') as csvfile:
+            csvfile.write(data)            
+    except Exception as e:
+        return e  #for debugging
```

Comment:
don't return exceptions: either don't catch them (then the caller will fail and log them), or catch and log yourself and return `False`